### PR TITLE
Add floating orb as a hover effect

### PR DESCRIPTION
**Pull request by robertkirsz**
I'm thinking about unifying our hover effects and came up with this little thing. What do you say? Not sure about its accessibility 🤔 

**What we have now:**
before

**What we can have:**
after

### CODE REVIEW

**Comment by robertkirsz** [comment]
I changed it so now all `<a />` have the Orb with an opt-out option via "no-orb" class. Will try to figure out how to handle mobile

File: .eslintrc.js [hunk 1]
```diff
@@ -11,9 +11,7 @@ module.exports = {
     indent: 'off',
     'comma-dangle': 'off',
     'space-before-function-paren': 'off',
-    'standard/object-curly-even-spacing': ['error', 'either'],
-    'standard/array-bracket-even-spacing': ['error', 'either'],
-    'standard/computed-property-even-spacing': ['error', 'even'],
+    'object-curly-spacing': 'off',
```
Comment (robertkirsz) [inline .eslintrc.js]: Prettier takes care of that

File: gatsby-config.js [hunk 2]
```diff
@@ -58,7 +58,7 @@ module.exports = {
         short_name: 'Expecto Frontendum',
         start_url: '/',
         background_color: '#ffffff',
-        theme_color: '#663399',
+        theme_color: '#005882',
```
Comment (robertkirsz) [inline gatsby-config.js]: Our blue

**Comment by kzadurska** [comment]
Oh well 🙈

**Comment by kzadurska** [comment]
oh well 🙈

**Pull request closed without merging**